test/graphql/types/Mutation/createChatMembership.test.ts

```
@@ -0,0 +1,604 @@
+} from "vitest";
+import { TalawaGraphQLError } from "~/src/utilities/TalawaGraphQLError";
+import { ChatMembershipResolver } from "../../../../src/graphql/types/Mutation/createChatMembership";
+
+type Fields = { [key: string]: any }; // Replace with actual field types
+type Operators = { eq: (field: string, value: any) => boolean };
+
+
+describe("ChatMembershipResolver", () => {
+
+	describe("creator", () => {
+		let parsedArgs = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'parsedArgs' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)